Index: ndhcpd/CMakeLists.txt
===================================================================
--- ndhcpd.orig/CMakeLists.txt
+++ ndhcpd/CMakeLists.txt
@@ -77,7 +77,7 @@ install(FILES ndhcpd-config.cmake DESTIN
 install(TARGETS ndhcpd-app
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT bin
  )
-install(FILES ndhcpd-app.log.properties DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/ndhcpd)
+install(FILES ndhcpd-app.log.properties DESTINATION /etc/ndhcpd)
 
 install(CODE "MESSAGE(\"To run service as non-root user the capabilities should be set using setcap(8) command.\")")
 install(CODE "MESSAGE(\"# /sbin/setcap cap_net_bind_service,cap_net_raw,cap_net_admin+epi ${CMAKE_INSTALL_PREFIX}/bin/ndhcpd\")")
